ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1234,7 +1234,7 @@ tests:
       OCP_ARCH: arm64
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-post-featureset
+    workflow: cucushift-installer-rehearse-azure-ipi-post-featureset-des
```

Comment:
So for now, the day2 enable des is only available in TP cluster?

Reply:
It's unrelated with TP, move out from TP job, add new job `azure-ipi-des-day2`, workflow is `cucushift-installer-rehearse-azure-ipi-disk-encryption-set-day2`